eng/Versions.props

```diff
@@ -6,7 +6,7 @@
     <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
     <PreReleaseVersionIteration>1</PreReleaseVersionIteration>
     <VersionPrefix>$(MajorVersion).$(MinorVersion).$(PatchVersion)</VersionPrefix>
-    <ApiCompatBaselineVersion>9.2.0</ApiCompatBaselineVersion>
+    <ApiCompatBaselineVersion>9.3.0</ApiCompatBaselineVersion>
     <ApiCompatBaselineSuffix>-preview.1.25105.6</ApiCompatBaselineSuffix>
```

Comment:
This needs to be updated as well

Reply:
Good catch, that said, why do we even check api compat for preview packages, that seems wrong. The main point of api compat is ensure we don't break API that has shipped stable.

Reply:
I see it as a guide to package authors that there are breaking changes. A "preview" package doesn't necessarily means breaking changes could be made as will.

Reply:
We don't necessarily want to break the build in that case though, right? I guess my point is that the Build shouldn't stop devs from changing APIs in a preview package, that is not an unnatural thing to happen. For now I've disabled it for non-stable packages but let me know if you feel strongly about this and I'm happy to discuss.

Reply:
The example is M.A.Testing - this is shipped as "stable" though it is still in a "dev" stage.

The M.E.AI packages are opted out of the compat validation.
https://github.com/dotnet/extensions/blob/64108d7d82a243f4ed8e1a983618a77c1294da1f/src/Libraries/Microsoft.Extensions.AI.Abstractions/Microsoft.Extensions.AI.Abstractions.csproj#L12

Reply:
> The example is M.A.Testing - this is shipped as "stable" though it is still in a "dev" stage.

Good catch, we should just fix that by marking the API as stabled in that case.

> The M.E.AI packages are opted out of the compat validation.

This is precisely what I want to avoid. That line was likely there to just disable Baseline validation (validate against an old version of the package) but that line is also opting out for in-package validation (as-in, making sure that frameworks shipped within the package are still validated and compatible).